File: software/examples/python/example_simple.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

HOST = "localhost"
PORT = 4223
UID = "XYZ" # Change XYZ to the UID of your Ozone Bricklet

from tinkerforge.ip_connection import IPConnection
from tinkerforge.bricklet_ozone import BrickletOzone

if __name__ == "__main__":
    ipcon = IPConnection() # Create IP connection
    o = BrickletOzone(UID, ipcon) # Create device object

    ipcon.connect(HOST, PORT) # Connect to brickd
    # Don't use device before ipcon is connected

    # Get current ozone concentration
    ozone_concentration = o.get_ozone_concentration()
    print("Ozone Concentration: " + str(ozone_concentration) + " ppb")

    raw_input("Press key to exit\n") # Use input() in Python 3
    ipcon.disconnect()

File: software/examples/java/ExampleSimple.java
import com.tinkerforge.IPConnection;
import com.tinkerforge.BrickletOzone;

public class ExampleSimple {
	private static final String HOST = "localhost";
	private static final int PORT = 4223;

	// Change XYZ to the UID of your Ozone Bricklet
	private static final String UID = "XYZ";

	// Note: To make the example code cleaner we do not handle exceptions. Exceptions
	//       you might normally want to catch are described in the documentation
	public static void main(String args[]) throws Exception {
		IPConnection ipcon = new IPConnection(); // Create IP connection
		BrickletOzone o = new BrickletOzone(UID, ipcon); // Create device object

		ipcon.connect(HOST, PORT); // Connect to brickd
		// Don't use device before ipcon is connected

		// Get current ozone concentration
		int ozoneConcentration = o.getOzoneConcentration(); // Can throw com.tinkerforge.TimeoutException
		System.out.println("Ozone Concentration: " + ozoneConcentration + " ppb");

		System.out.println("Press key to exit"); System.in.read();
		ipcon.disconnect();
	}
}
